app/api/entities/reaction_entity.rb

```diff
@@ -46,6 +46,7 @@ class ReactionEntity < ApplicationEntity
       expose! :tlc_description,                             unless: :displayed_in_list
       expose! :tlc_solvents,                                unless: :displayed_in_list
       expose! :variations,            anonymize_with: [],                               using: 'Entities::ReactionVariationEntity'
+      expose! :vessel_size
```

Comment:
Since `vessel_size` is not a scalar, maybe make the definition more specific by defining a separate entity?
E.g., `using: Entities::VesselSize`.

Reply:
We agreed to limit the creation of entities to the model and skip creation of entities for nested data structures.